luminoso_api/v5_cli.py

```diff
@@ -68,10 +69,67 @@ def _read_params(input_file, json_body, p_params):
     return params
 
 
+def connect_with_token_args(args):
```

Comment:
That someone is me, a frequent user of these scripts. I found the previous workflow for setting up a token to be error-prone and unclear.

I am confused by your note here, because you seem to be describing what I changed it _to_. The new version tells you how to get a token (from the appropriate location for the API server you're connecting to), and then once you supply it, offers to save it.

Reply:
I meant modifying the actual errors thrown in the client to have more informative messages, rather than implementing this entire new interface that has a lot of problems.